images-list

```diff
@@ -3,4 +3,12 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+grafana/grafana rancher/grafana-grafana 7.1.5
+istio/kubectl rancher/istio-kubectl 1.5.10
 library/busybox rancher/library-busybox 1.31.1
+library/nginx rancher/nginx 1.19.2-alpine
```

Comment:
```suggestion
library/nginx rancher/library-nginx 1.19.2-alpine
```

Reply:
@brandond Both nginx and thanos were in image-list with these names. I can rename them, but they would go into a new repository. Is that okay?

Reply:
@brandond I'll rename them, but new repositories are required since they were previously in image-list with these names.

Reply:
We'll also need to update system-charts for new image name. @aiyengar2 this may affect monitoring v2 going forward.

Reply:
@PennyScissors do you have access to create the new repos, or do we need to pull in someone else to do that? I don't personally have access to the rancher org.

Reply:
We don't use an nginx image afaik. @brendarearden are you using it for the broken magic links stuff?

Reply:
@aiyengar2 I was able to prove out that it works to use the built in url workarounds, so I don't plan on using nginx.

Reply:
@aiyengar2 What does that mean exactly? Nginx is used here: https://github.com/rancher/system-charts/blob/7f97198bdd7cd39b87571a58fd5cde4f124bc49c/charts/rancher-monitoring/v0.1.2/values.yaml#L262

Reply:
Monitoring V1 uses it because it uses an authentication proxy in front of Grafana / Prometheus / Alertmanager in order to enforce Rancher ECM RBAC for Projects for users trying to access the UIs for those components.

Monitoring V2 does not utilize this since it doesn't need to support project-scoped monitoring